bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -69,6 +73,13 @@ public class FormContainerImpl extends AbstractContainerImpl implements FormCont
     private static final String FD_SCHEMA_REF = "fd:schemaRef";
     public static final String FD_FORM_DATA_ENABLED = "fd:formDataEnabled";
 
+    private static final String CUSTOM_FUNCTION_CONFIG_BUCKET_NAME = "settings/cloudconfigs";
```

Comment:
configuration-related logic can be managed independently in a separate utility, making this code more organized and easier to maintain, this way sling model remains focused on its main purpose.

```
 private ConfigurationManager configManager = ConfigurationManager.getInstance();
 configManager.getExternalCustomFunctionUrl(configurationResourceResolver);
``

Reply:
done, implemented as a slingModel after discussion with @rismehta

Reply:
removed it from the form container model